Example/AztecUITests/HighPriorityIssuesTests.swift

```diff
@@ -0,0 +1,147 @@
+import XCTest
+
+class HighPriorityIssuesTests: XCTestCase {
+        
+    private var app: XCUIApplication!
+    private var richTextField: XCUIElement!
+    
+    override func setUp() {
+        super.setUp()
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIDevice.shared().orientation = .portrait
+        app = XCUIApplication()
+        app.launch()
+        
+        let blogsPage = BlogsPage.init(appInstance: app)
+        blogsPage.gotoEmptyDemo()
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    // Github issue https://github.com/wordpress-mobile/AztecEditor-iOS/issues/385
+    func testLongTitle() {
```

Comment:
This test is still failing on larger devices (e.g. iPad Pro). If there isn't a way to get it to work for both large and small devices, could we restrict it to only run when testing smaller devices?

Reply:
I don't think its possible to tag which tests to run with current test runner (as it's done in wp-e2e-tests). The only solution I see is to move such tests to separate files and run them only for small screens. And in general, I think it would be better to address this issue once we'll have them running on multiple devices.

Reply:
I looked around and found [these hints and tips](http://averagepro.com/2015/11/06/xctest-ui-testing-hints-and-tips/) that include conditional code for testing on iPhone v. iPad. I updated the example shown there and got it to work in this test like this:

```
if (app.windows.element(boundBy: 0).horizontalSizeClass == .compact || app.windows.element(boundBy: 0).verticalSizeClass == .compact) {
            titleTextView.typeText("very very very very very very long title in a galaxy not so far away")
        } else {
            titleTextView.typeText("very very very very very very long title in a galaxy not so far away very very very very very very long title in a galaxy not so far away")
        }
```